src/standard-live-connect.js

```diff
@@ -130,14 +132,15 @@ function _standardInitialization (liveConnectConfig, externalStorageHandler, ext
     eventBus.init()
     const callHandler = CallHandler(externalCallHandler)
     errorHandler.register(liveConnectConfig, callHandler)
-
-    const storageHandler = StorageHandler(liveConnectConfig.storageStrategy, externalStorageHandler)
+    const storageStrategy = liveConnectConfig.gdprApplies ? StorageStrategy.disabled : liveConnectConfig.storageStrategy
```

Comment:
We probably need to set the new parameters explicitly here when `gdprApplies` is true